srml/session/src/lib.rs

```diff
@@ -117,7 +117,7 @@ decl_storage! {
 	trait Store for Module<T: Trait> as Session {
 
 		/// The current set of validators.
-		pub Validators get(validators) config(): Vec<T::AccountId>;
+		pub Validators get(validators) config(): Vec<(T::AccountId, u64)>;
```

Comment:
Am I right that this change is a non-backward compatible, which means a chain reset is required to apply this change? Because the new runtime won't able to decode the old data that was encoded with old format.

Reply:
you are correct, yes.

Reply:
a workaround would be a new item `map T::AccountId => u64`. alternatively, you could introduce a new `grandpa-session` module mostly copy-pasted from `session` module but with any grandpa alterations.